data/raw/USB6014.txt

```diff
@@ -0,0 +1,158 @@
+1	2	277.1
```

Comment:
We should ask Scott to have some sort of doc on this, the more we add networks beyond the basics it'll be hard to distinguish them in 1-2 years from now.